docs/src/pages/components/selectfield/react.mdx

```diff
@@ -55,6 +56,22 @@ To manually control the SelectField state, you can use the `value` and `onChange
   </ExampleCode>
 </Example>
 
+### Options
+
+To create a simple SelectField in which each `<option>` has matching `value`, `label` and `children`, pass a `string[]` to the `options` prop.
```

Comment:
Maybe leave the type information out of the docs, in case it changes and doesn't stay in sync. 
```suggestion
Create a simple `SelectField` by passing a list to the `options` prop.
```